### ISSUE TITLE
Dynamic Data on the Map View

### ISSUE THREAD

**Issue by keenanjohnson** [Ribbit-Network/ribbit-network-dashboard#10]
The data view for each sensor point on the global map should have labels (Lat, Lon, DataType) in preparation for additional GHGs.

**Comment by brianstrauch** [comment]
How is this different from #9?

**Comment by keenanjohnson** [comment]
Oh whoops. I copied the wrong description here it looks like. Let me resolve that

**Comment by brianstrauch** [comment]
I think we should support two modes:
1. Live mode, where the map is constantly updating with the newest data
2. A user-defined time range (i.e. last 1hr, or a range like Jan-Feb), where the map and graph are static

In both cases, there should definitely only be one query.

**Comment by keenanjohnson** [comment]
Yeah that sounds great to me! I wonder if the live updating mode is actually useful or interesting? Probably I suspect?

**Comment by brianstrauch** [comment]
The live map would probably just make the website interesting to visit haha
